pkg/orchestrator/gitHubActions.go

```diff
@@ -1,79 +1,116 @@
 package orchestrator
 
 import (
-	"os"
-	"strings"
+	"fmt"
+	piperHttp "github.com/SAP/jenkins-library/pkg/http"
 	"time"
 
 	"github.com/SAP/jenkins-library/pkg/log"
 )
 
-type GitHubActionsConfigProvider struct{}
+type GitHubActionsConfigProvider struct {
+	run run
+}
+
+func (g *GitHubActionsConfigProvider) initOrchestratorProvider(settings *OrchestratorSettings) error {
+	client := piperHttp.Client{}
+	client.SetOptions(piperHttp.ClientOptions{
+		Password:         settings.GitHubToken,
+		MaxRetries:       3,
+		TransportTimeout: time.Second * 10,
+	})
+	ghURL := getEnv("GITHUB_URL", "")
+	switch ghURL {
+	case "https://github.com/":
+		ghURL = "https://api.github.com/"
+	default:
+		ghURL += "api/v3/"
+	}
+	resp, err := client.GetRequest(
+		fmt.Sprintf(
+			"%s/repos/%s/actions/runs/%s", ghURL, getEnv("GITHUB_REPOSITORY", ""), getEnv("GITHUB_RUN_ID", ""),
+		),
+		map[string][]string{
+			"Accept":        {"application/vnd.github+json"},
+			"Authorization": {"Bearer $GITHUB_TOKEN"},
+		}, nil)
+	if err != nil {
+		return fmt.Errorf("can't get API data: %w", err)
 
-func (g *GitHubActionsConfigProvider) InitOrchestratorProvider(settings *OrchestratorSettings) {
-	log.Entry().Debug("Successfully initialized GitHubActions config provider")
+	}
```

Comment:
Would it be fine to move this to `github` pkg?

Reply:
Not sure. It's a specified case to the telemetry with gh actions environment.